src/components/pagination/bl-pagination.ts

```diff
@@ -80,10 +61,28 @@ export default class BlPagination extends LitElement {
   selectLabel = 'Show';
 
   /**
-   *  Sets the option texts.
+   * Sets the items per page options of the select element
+   *  PROPERTY
    */
-  @property({ attribute: 'option-text', type: String })
-  optionText = 'Items';
+  @property({ type: Array })
```

Comment:
```suggestion
  @property({ type: Array, attribute: false })
```

Use `attribute: false` to use this as a property only. Otherwise still component will try to parse this as an attribute.